src/gameui/LuaFace.cpp

```diff
@@ -9,39 +9,59 @@ namespace GameUI {
 
 	class LuaFace {
 	public:
-		static inline Uint32 _unpack_flags(lua_State *l)
+		static inline bool _get_int(lua_State *l, int table, const char *key, int &output)
```

Comment:
NOTE: Instead of writing a new function (which I think should have a name more like "_find_int" if I guess correctly its complexity), may it be better if you find a way to use existing code:

https://github.com/pioneerspacesim/pioneer/blob/1847087e21a43e3675b066793639c09caef8ce60/src/LuaTable.h#L46-L59

...It seems it do the same without reinventing the wheel

Reply:
Oh, good point. Still getting familiar with the code, and haven't come across that class, thanks!

Just pushed another commit. I'll squash them all later once it's good to merge.

Reply:
My 2 cents are that this should go as is: changing the overall design cost
time and is not worth the effort... It should be better done when (and if :( )
this will be rewritten in ImGui.

Reply:
> My 2 cents are that this should go as is

Right, I meant it in the "if nothing else comes up"... like the using #include with quote marks thing ;)